src/applications/gi/tests/e2e/00-required.cypress.spec.js

```diff
@@ -0,0 +1,69 @@
+import { initMockProfile, expectLocation } from './cypress-helpers';
+import institutionProfile from '../data/institution-profile.json';
+import searchResults from '../data/search-results.json';
+import { createId } from '../../utils/helpers';
+import {
+  typeOfInstitution,
+  search,
+  selectSearchResult,
+  displayLearnMoreModal,
+  collapseExpandAccordion,
+} from './gi-helpers';
+
+const institutionAttributes = institutionProfile.data.attributes;
+
+describe('Institution', () => {
+  beforeEach(() => {
+    cy.route('/v0/gi/institutions/', searchResults);
+
+    initMockProfile(institutionProfile);
+    cy.visit('/gi-bill-comparison-tool').injectAxe();
+    cy.axeCheck();
+  });
+
+  it('Default institution profile flow with giBillChapter chapter 33', () => {
+    // Landing Page
+    typeOfInstitution('school');
+
+    cy.axeCheck();
+    cy
+      .get('.keyword-search input[type="text"]')
+      .type(institutionAttributes.name).type;
+    search();
+    expectLocation('/search');
+
+    // Search Page
+    searchResults.data.forEach(result => {
+      const resultId = `#search-result-${result.attributes.facility_code}`;
+      cy.log(resultId);
+      cy.get(createId(resultId)).should('be.visible');
+    });
+
+    const profileLink = `/profile/${
+      searchResults.data[0].attributes.facility_code
+    }`;
+
+    selectSearchResult(profileLink);
+
+    // Profile Page
+    displayLearnMoreModal();
+
+    // Estimate your benefits
+    collapseExpandAccordion('Estimate your benefits');
+
```

Comment:
Missing the equivalent of below in this test

```
  GiHelpers.checkSectionAccordion(client, false, 'yourMilitaryDetails');	
  GiHelpers.checkSectionAccordion(client, true, 'schoolCostsAndCalendar');	
  GiHelpers.checkSectionAccordion(client, true, 'learningFormat');	
  GiHelpers.checkSectionAccordion(	
    client,	
    true,	
    'scholarshipsAndOtherVAFunding',	
  );
```

Reply:
`gi-helpers.js` will need the cypress versions of the `checkSectionAccordion` function